mffpy/bin_writer.py

```diff
@@ -138,3 +138,16 @@ def write(self, filename: str, *args, **kwargs):
             num_written = fo.write(byts)
         assert num_written == len(byts), f"""
         Wrote {num_written} bytes (expected {len(byts)})"""
+
+class StreamingBinWriter(BinWriter):
+
+    """
+    Subclass of BinWriter to support streaming bin file to disk.
+    """
+
+    def __init__(self, sampling_rate: int, mffdir: str, data_type: str = 'EEG'):
```

Comment:
Are then any limitations / expectations for `sampling_rate`, `mffdir`, or `data_type` that are non-obvious from the casual read?

(i.e. We don't support smb file paths or some such)

Reply:
I've added some comments. 

Specifically, as regards the mffdir value, the directory must exist before the object is created, and this is noted. As far as limitations to where the folder can exist: this utilizes the same underlying functionality as the builtin open() and so has the same characteristics. These characteristics are shared by the BinWriter class as it uses open() directly to write the file.